packages/app/components/Nav/NavBar.tsx

```diff
@@ -123,7 +123,6 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
   const [visible, setVisible] = useState<boolean>(false);
   const { pathname } = useRouter();
   const { course } = useCourse(courseId);
-  const role = useRoleInCourse(courseId);
```

Comment:
This I think we can leave in cause I had previously pushed a hot fix to `data-insights-WIP` branch

Reply:
Does that mean remove it? or just keep it as is? (because i am afraid if I remove this, then the cypress tests might not run correctly)